modules/images/webp-uploads/load.php

```diff
@@ -742,3 +742,29 @@ function webp_uploads_get_image_sizes_additional_mime_type_support() {
 
 	return $allowed_sizes;
 }
+
+/**
+ * Updates the quality of images used in the wp editor.
+ * For WP version less than 6.1 or WebP returns 82,
+ * Otherwise returns default quality.
```

Comment:
```suggestion
 * Updates the quality of WebP image sizes generated by WordPress to 82.
 *
 * Prior to WordPress 6.1, the MIME type was not correctly provided to this filter, so for those versions this function
 * simply returns 82 universally (since all other MIME types already use 82 anyway).
```